packages/dev/s2-docs/src/SearchResultsMenu.tsx

```diff
@@ -30,7 +30,7 @@ interface SearchResultsMenuProps {
   searchRef: React.RefObject<TextFieldRef<HTMLInputElement> | null>,
   showCards: boolean,
   renderCardList: () => React.ReactNode,
-  filter?: AutocompleteProps['filter'],
+  filter?: AutocompleteProps<any>['filter'],
```

Comment:
Wouldn't this be `AutocompleteProps<ComponentItem>`?
or is it `AutocompleteProps<ComponentItem | ComponentSection>`